R/shiny.R

```diff
@@ -379,7 +380,9 @@ sessionDataURL = function(session, data, id, filter) {
     res = tryCatch(filter(data, params), error = function(e) {
       list(error = as.character(e))
     })
-    httpResponse(200, 'application/json', enc2utf8(toJSON(res, dataframe = 'rows')))
+
+    jsonArgs = c(list(x = res, dataframe = 'rows'), attr(data, "TOJSON_ARGS", exact = TRUE))
+    httpResponse(200, 'application/json', enc2utf8(do.call(toJSON, jsonArgs)))
```

Comment:
I only want to keep these two lines in this PR, and change `attr(data, "TOJSON_ARGS", exact = TRUE)` to `getOption('htmlwidgets.TOJSON_ARGS')`, i.e. only support the global option `htmlwidgets.TOJSON_ARGS`.

Reply:
Yes, I feel the same, too. How about an option in DT? I’m afraid if the user uses another HTML widgets, the global option may lead to undesirable results on the other widget. What’s your option?

Reply:
Let's consider that in the future when users complain about it :) I don't think many people actually know this option at all. In my eyes, it is a quite trivial matter.